source/fundamentals/indexes.txt

```diff
@@ -230,6 +230,8 @@ Multikey indexes behave differently from non-multikey indexes in terms of query
 sort behavior. For a full explanation of multikey indexes, including a discussion of their behavior and limitations,
 refer to the :manual:`Multikey Indexes page </core/index-multikey>` in the MongoDB manual.
 
+.. _textIndexes:
```

Comment:
Nitpick:
The other anchors use snake case (words separated by a dash). It might be better to stay consistent with that rather than use camelcase.